### ISSUE TITLE
Interact with CHANGELOG outside of directory

### ISSUE THREAD

**Issue by andrewcstewart** [mc706/changelog-cli#39]
It would be nice to be able to interact with a CHANGELOG.md when not in the same directory as the file (for example, if you are a sub-directory, etc).  

Ideas:

- Accept `--path` argument to `CHANGELOG.md` location.
- Accept `$CHANGELOG=/path/to` environment variable.
- Traverse the chain of parent directories to identify a `CHANGELOG.md` file.

**Comment by KUGA2** [comment]
Yes please. In my case the file is `changelog.md` not `CHANGELOG.md` therefore it does not work.
